Dockerfile

```diff
@@ -20,4 +20,4 @@ RUN apk --no-cache add dumb-init && rm -rf /var/cache/apk/*
 WORKDIR /app
 COPY --from=yarn-prod /app ./
 EXPOSE 80
-CMD ["dumb-init", "node", "index.js"]
+CMD ["dumb-init", "node", "src/index.js"]
```

Comment:
This is wrongly inserted. 'src' folder doesn't exists and index.js is copied from dist folder created by yarn build to the /app folder which is CWD

Reply:
But the downside from my change up is that we're including scripts folder for both volume-tracker and gas-refund builds while we needed only for gas-refund build. 
So I re-worked the thing so that volume-tracker bundle is untouched but gas-refund is now bringing both src+scripts folders (we need some code from src). Done in [this commit](https://github.com/paraswap/paraswap-volume-tracker/pull/24/files)